data/en/test_confirmation_question.json

```diff
@@ -44,7 +44,21 @@
                         "default": 0
                     }],
                     "id": "number-of-employees-total-question",
-                    "title": "How many employees work at {{ first_non_empty_item(metadata['trad_as'], metadata['ru_name']) }}?",
+                    "title": {
+                        "text": "How many employees work at {company_name}?",
+                        "placeholders": [{
+                            "placeholder": "company_name",
+                            "transforms": [{
+                                "transform": "first_non_empty_item",
+                                "arguments": {
+                                    "items": {
+                                        "source": "metadata",
+                                        "identifier": ["trad_as", "ru_name"]
+                                    }
+                                }
+                            }]
+                        }]
+                    },
```

Comment:
We should consider whether we should have piping in test surveys that don't need it, we should always be creating the minimal schema to test a feature.